tests/communication/ros2/test_api.py

```
@@ -14,121 +14,26 @@
-    time.sleep(0.5)
-    # Then join threads with a timeout
-    for thread in threads:
-        thread.join(timeout=2.0)
-
-
-@pytest.fixture(scope="function")
-def ros_setup() -> Generator[None, None, None]:
-    rclpy.init()
-    yield
-    rclpy.shutdown()
+ros_setup  # type: ignore prevents pytest from complaining about unused fixture
```

Ruff (0.8.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)